code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -289,11 +381,28 @@
 	results = list("slimejelly" = 1)
 	required_reagents = list("toxin" = 1, "jellyblood" = 1)
 
-/datum/chemical_reaction/corazone
+/datum/chemical_reaction/corazone //7.433 -> 12.5 (should go slightly above 12.5)
 	name = "Corazone"
 	id = "corazone"
 	results = list("corazone" = 3)
 	required_reagents = list("phenol" = 2, "lithium" = 1)
+	mix_message = "The solution gurgles as it's viscosity increases."
```

Comment:
"its" is the possessive form of "it", not "it's"

Reply:
pesky!